src/api/audio.c

```diff
@@ -68,8 +68,14 @@ int l_lovrAudioNewSource(lua_State* L) {
     stream = *type;
   } else {
     Blob* blob = luax_readblob(L, 1, "Source");
+    if (!blob) // readblob responsible for setting Lua error
+      return 0;
```

Comment:
I don't see how this is ever true?  Looking at readblob, it should either return a Blob or throw an error.

Reply:
So I guess the correct thing to check is if (!blob->data) ?